cypress/integration/9-room-status-bar/account-suspension.ts

```diff
@@ -0,0 +1,113 @@
+/*
+Copyright 2022 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/// <reference types="cypress" />
+
+import { SynapseInstance } from "../../plugins/synapsedocker";
+
+describe("Room Status Bar", () => {
+    const userAccountSuspended = 'ORG.MATRIX.MSC3823.USER_ACCOUNT_SUSPENDED';
+    const href = "http://example.org";
+    const text = "Hello, world";
+
+    let synapse: SynapseInstance;
+    let roomId: string;
+
+    beforeEach(() => {
+        cy.on('uncaught:exception', () => {
+            // We simulate server errors. These are (sometimes?) considered as uncaught
+            // exceptions, which make the test fail by default.
+            // We don't want that to happen.
+            return false;
+        });
+
+        // Start a Synapse, create a user, create a room.
+        return cy.startSynapse("default").then(data => {
+            synapse = data;
+            return cy.initTestUser(synapse, "Alice");
+        }).then(_ => {
+            return cy.createRoom({});
+        }).then(data => {
+            roomId = data;
+            cy.visit("/#/room/" + roomId);
+        });
+    });
+
+    afterEach(() => {
+        // Attempting to stop Synapse breaks the tests.
+        // Not sure what's going on here.
+        // cy.stopSynapse(synapse);
+    });
+
+    it("shouldn't display an error message when there is no error", () => {
+        // User sends message
+        cy.get(".mx_RoomView_body .mx_BasicMessageComposer_input").type(`${text}{enter}`);
+
+        // Wait for message to send
+        cy.get(".mx_RoomView_body .mx_EventTile").contains(".mx_EventTile[data-scroll-tokens]", text);
+
+        // Give an error a little time to show up. It shouldn't.
+        cy.wait(1_000);
```

Comment:
we don't use this syntax:
```suggestion
        cy.wait(1000);
```

Reply:
Will do, but may I suggest amending this policy? 16_777_384 is much more readable than 16777384.

Reply:
I personally disagree on that, tbh :p 

I'm not sure what others on the team think, but I suspect since we haven't done so up until now we aren't likely to remember to do it, so will end up fighting the linter instead of working with it.